lib/README.md

```diff
@@ -356,11 +356,34 @@ docs:
       returns: Promise<number>
       static: true
 
+    - name: fold
+      since: unreleased
+      desc: |
+        Folds an array into a promise or a value. The array is traversed sequentially and
+
+        The reducer function can return a promise or directly a value. It always received the last resolved value.
+
+        The folding will stop at the first rejection encountered.
+        ```
```

Comment:
There's no example here, but this looks like the start of a code block. Could you include one?